2024/week2_more_crypto_snarks_starks.md

```diff
@@ -79,6 +79,18 @@ Groth16 is one of the famous zkSNARKs. Just remember that its biggest advantage
 STARKs are a type of zero-knowledge proof system that is post-quantum secure. It usually involves larger proof size and faster proof size. There are two articles written by Vitalik that explain STARKs in detail. You can read them here:
 1. https://vitalik.eth.limo/general/2017/11/09/starks_part_1.html
 2. https://vitalik.eth.limo/general/2017/11/22/starks_part_2.html
+3. https://vitalik.eth.limo/general/2018/07/21/starks_part_3.html
+#### Stark Anatomy 
+
+This series of articles is a six part tutorial explaining the mechanics of the STARK proof system. It is directed towards a technically-inclined audience with knowledge of basic maths and programming.
+https://aszepieniec.github.io/stark-anatomy/
+
+#### Exploring circle STARKs
+
+Introduction to Circle STARKs, an optimization that makes STARK proofs more efficient and scalable using structured polynomial commitme
```

Comment:
I think this is getting too deep for a week 2. Happy to include it and mark it as optional though. Also, I think you forgot to finish typing the sentence.